ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -24,10 +27,23 @@ def Acosh(*, x, name="Acosh"):
     return ivy.acosh(x)
 
 
-Add = tf_frontend.math.add
+def Add(*, x, y, name="Add"):
+    return map_raw_ops_alias(
+        tf_frontend.math.add,
+        x=x,
+        y=y,
+        name=name,
+    )
 
 
-ArgMax = tf_frontend.math.argmax
```

Comment:
This can then just be `ArgMax = map_raw_ops_alias(tf_frontend.math.argmax, kwargs_to_replace={'dimension': 'axis'})`. Again, feel free to decide the name of this argument, `kwargs_to_replace` is just an example.